[Export] Add math module for deserialization

henryhu6 opened the pull request:
Summary: As title

Test Plan: ci

Differential Revision: D75580646

henrylhtsang commented:
linter

yushangdi reviewed (approved):
LGTM, please fix the linter error

henryhu6 committed:
[Export] Add math module for deserialization (#154643)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/154643

As title

Test Plan: ci

Reviewed By: henryoier, yushangdi

Differential Revision: D75580646

izaitsevfb commented:
@pytorchbot merge -i
